# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1711561402 -3600
# Parent  487ddcf49644b1eec0be71c7896a4ac93f542b54
Misc: Don't allow language pack download from Thunderbird site.
* * *
Misc: Allow language pack download from Betterbird site.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -872,18 +872,12 @@ pref("intl.regional_prefs.use_os_locales
 //      it for beta.
 //  *.liveReload - Switching a langpack will change the language without a restart.
 //  *.liveReloadBidirectional - Allows switching when moving between LTR and RTL
 //      languages without a full restart.
-pref("intl.multilingual.enabled", true);
-#if defined(RELEASE_OR_BETA)
+pref("intl.multilingual.enabled", true); // Allow manual or distro installation of langpacks.
 pref("intl.multilingual.downloadEnabled", true);
 pref("intl.multilingual.liveReload", false);
 pref("intl.multilingual.liveReloadBidirectional", false);
-#else
-pref("intl.multilingual.downloadEnabled", false);
-pref("intl.multilingual.liveReload", false);
-pref("intl.multilingual.liveReloadBidirectional", false);
-#endif
 
 // if true, use full page zoom instead of text zoom
 pref("browser.zoom.full", true);
 
diff --git a/mail/components/preferences/messengerLanguages.js b/mail/components/preferences/messengerLanguages.js
--- a/mail/components/preferences/messengerLanguages.js
+++ b/mail/components/preferences/messengerLanguages.js
@@ -435,9 +435,9 @@ var gMessengerLanguagesDialog = {
       onSelect: item => this.availableLanguageSelected(item),
       onChange: item => {
         this.hideError();
         if (item.value == "search") {
-          this.loadLocalesFromAMO();
+          window.top.openContentTab("https://www.betterbird.eu/downloads", "tab");
         }
       },
     });
 
@@ -447,9 +447,10 @@ var gMessengerLanguagesDialog = {
 
     // If the user opened this from the "Search for more languages" option,
     // search AMO for available locales.
     if (search) {
-      return this.loadLocalesFromAMO();
+      window.top.openContentTab("https://www.betterbird.eu/downloads", "tab");
+      return undefined;
     }
 
     return undefined;
   },
